DOCS/man/options.rst

```diff
@@ -2501,7 +2501,7 @@ Subtitles
 
         Using this option may lead to incorrect subtitle rendering.
 
-``--sub-ass-override=<yes|no|force|scale|strip>``
+``--sub-ass-override=<no|yes|scale|force|strip>``
```

Comment:
This same change to the order of the possible values should also be applied to the documentation for `secondary-sub-ass-override`.

Reply:
I didn't even know we documented `secondary-*` options like that, that's kinda stupid